features/admin.users.v1/components/user-profile.tsx

```diff
@@ -102,6 +107,7 @@ import {
     isMultipleEmailsAndMobileNumbersEnabled,
     isSchemaReadOnly
 } from "../utils/user-management-utils";
+import { log } from "console";
```

Comment:
This is unnecessary, isn't it?

Reply:
82d4ff507d55ec7ed6c0b57080b6e89963594857